packages/tidy/src/sequences/fullSeq.ts

```diff
@@ -33,29 +33,17 @@ export function vectorSeqDate(
     sequence.push(new Date(value));
 
     // increment date
-    if (
-      granularity === 'day' ||
-      granularity === 'd' ||
-      granularity === 'days'
-    ) {
+    if (['second', 's', 'seconds'].includes(granularity)) {
+      value.setUTCSeconds(value.getUTCSeconds() + 1 * period);
+    } else if (['minute', 'min', 'minutes'].includes(granularity)) {
+      value.setUTCMinutes(value.getUTCMinutes() + 1 * period);
+    } else if (['day', 'd', 'days'].includes(granularity)) {
```

Comment:
Can you revert this back to the long form with the `||`s? I'm not really sure if the perf difference will ever matter, but I don't see the point in allocating an array and calling a function a bunch of times while cascading through the if statement. The minifier will shrink it well enough. One day js will just provide an `in` operator... Maybe the js optimizer is smart enough to do it already, I'm not sure.

Reply:
Yes, I will do. Actually it was not even for the performance. I was battling between `less lines` and `readability`. Guess the less lines argument won at that time.